src/components/editor/EditorPane.tsx

```diff
@@ -112,6 +115,16 @@ export class EditorPane extends React.Component<EditorPaneProps, {
             onClick={() => {
               return this.props.onSplitEditor && this.props.onSplitEditor();
             }}
+          />,
+          <Button
+            key="save"
+            icon={<GoClippy />}
+            label="Save"
+            title="Save: CtrlCmd + S"
+            isDisabled={this.editor && (!this.editor.props.view.file.isDirty)}
+            onClick={() => {
+              this.editor.monaco.editor.trigger("anystring", "save", "");
```

Comment:
You should be able to just write `this.props.file.save();`. Note that we are now in the process of moving to a Flux architecture (see the actions in `AppActions.ts`). Under the new design, we would create a `SAVE_FILE` action and replace calls to `file.save()` with a `saveFile(file)` which would then dispatch a `SAVE_FILE` action. I think for now it's okay to land a usage of `file.save()` and we can follow up in another PR with a fix for the Flux thing.